test/unit/test_dss_client.py

```diff
@@ -82,10 +88,12 @@ def _fake_do_download_file_with_barrier(*args, **kwargs):
     fh.write(b'Here we write some stuff so that the fake download takes some time. '
              b'This helps ensure that multiple threads are writing at once and thus '
              b'allows us to test for race conditions.')
+    time.sleep(random.random())  # sleep for some small random amount of time
```

Comment:
Comment is obvious. I think the string above explains the intent sufficiently.

Reply:
Absolutely agree.